src/gpuarray/buffer.h

```diff
@@ -746,10 +746,6 @@ typedef enum _ga_usefl {
   /* If you add a new flag, don't forget to update both
      gpuarray_buffer_{cuda,opencl}.c with the implementation of your flag */
   /**
-   * The kernel is made of PTX code.
-   */
-  GA_USE_PTX =      0x1000,
```

Comment:
Fine, I've added it.  But I still feel like this is an interface violation of the documentation.